templates/cube/microcerts.html

```diff
@@ -0,0 +1,379 @@
+{% extends "cube/base_cube.html" %}
+
+{% block title %} Microcerts | CUBE {% endblock %}
+
+{% block meta_description %}This is the CUBE Microcerts page{% endblock meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1IHmr2SIxcEKVNN2sA-z_32PV0fCXi38pmeVP7k7O1mM/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--square-suru has-cube is-dark is-slanted--bottom-left">
+  <div class="row u-equal-height u-vertically-center">
+    <div id="hero-content" class="col-6">
+      {% if certified_badge %}
+
+        <h1>CUBE 2020 complete</h1>
+        <p class="p-heading--4">You have completed all 15 microcerts, proving your Ubuntu expertise. You are now a Certified Ubuntu Engineer.</p>
+
+      {% elif has_enrollments %}
+
+      <h1>Your road to CUBE</h1>
+        {% if passed_courses > 0 %}
+        <p class="p-heading--4">You have completed {{ passed_courses }} microcerts.</br>Complete {{ (15 - passed_courses) }} more microcerts to attain CUBE.</p>
```

Comment:
Can this value be calculated in the backend as the number of courses could change?